java/jdbc/SpringSharding/JDBCTemplate/README.md

```diff
@@ -0,0 +1,78 @@
+## Overview
+
+This project demonstrates the use of the latest sharding [feature](https://github.com/spring-projects/spring-framework/pull/31506) in Spring Framework with the Oracle Database.
+The feature is about supporting direct routing to sharded databases.
+
+This version uses Spring Data JDBC (JdbcTemplate), for data access.
+
+You can use the datasource configurations provided in this project as a template for setting up the sharding feature in your own projects.
+
+## Configuration
+
+### Database
+
+You can refer to the [Oracle Docs](https://docs.oracle.com/en/database/oracle/oracle-database/21/shard/sharding-deployment.html#GUID-F99B8742-4089-4E77-87D4-4691EA932207)
+to learn how to set up and deploy an Oracle sharded database.
+You can also refer to [Oracle Database Operator](https://github.com/oracle/oracle-database-operator) that makes deploying a sharded database on a Kubernetes Cluster an easy process.
+
+After your sharded database is set, run the following SQL script on the shard catalog to create your tables.
```

Comment:
As what database user are you supposed to run this script? There is no indication of that. Is it `SYSTEM` or the shard director user, or the catalog user, or are these all the same the user? It would probably help the user to be more specific on how to execute these DDLs and what user(s) to use in the properties.

Reply:
After your sharded database is set, connect to the shard catalog as sysdba and create the demo application schema user.